workflows/run.nf

```diff
@@ -40,25 +41,14 @@ workflow RUN {
         }
     }
 
-    // Prepare samplesheet
-    if ( params.grouping ) {
-        samplesheet = Channel
-            .fromPath(params.sample_sheet)
-            .splitCsv(header: true)
-            .map { row -> tuple(row.sample, file(row.fastq_1), file(row.fastq_2), row.group) }
-        samplesheet_ch = samplesheet.map { sample, read1, read2, group -> tuple(sample, [read1, read2]) }
-        group_ch = samplesheet.map { sample, read1, read2, group -> tuple(sample, group) }
-    } else {
-        samplesheet = Channel
-            .fromPath(params.sample_sheet)
-            .splitCsv(header: true)
-            .map { row -> tuple(row.sample, file(row.fastq_1), file(row.fastq_2)) }
-        samplesheet_ch = samplesheet.map { sample, read1, read2 -> tuple(sample, [read1, read2]) }
-        group_ch = Channel.empty()
-    }
+    // Load samplesheet
```

Comment:
I think I would also put the grouping check in here?

Reply:
Done.